app/src/main/java/com/abhijitvalluri/android/fitnotifications/AppSettingsActivity.java

```diff
@@ -130,7 +133,8 @@ public void onClick(View v) {
                                                                            mStopTimeMinute,
                                                                            mStartTimeHour,
                                                                            mStartTimeMinute,
-                                                                           R.string.stop_time_heading,
+                                                                           startTimeFormatted,
```

Comment:
Fix this as well. Again, you may either remove this and not show the time in the toast or update the OnClickListener each time you set the start/stop time.